src/parser/core/modules/AoEUsages.tsx

```diff
@@ -8,16 +8,16 @@ import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Sugge
 import React from 'react'
 import {Table} from 'semantic-ui-react'
 
-export interface AoeAbility {
+export interface AoEAction {
 	/**
-	 * The AoE ability to check usages of.
+	 * The AoE action to check usages of.
 	 */
-	aoeAbility: Action
+	AoEAction: Action
```

Comment:
Hypernit, we mostly use snake-case for these kinda things, if you look at other core modules they're all like `aoeAction`. My initial comment about the E was for the interface title where we case appropriately. Not a big deal, just a consistency thing.